tools/serverpod_cli/lib/src/analyzer/dart/endpoint_analyzers/endpoint_parameter_analyzer.dart

```diff
@@ -44,6 +45,43 @@ abstract class EndpointParameterAnalyzer {
   ) {
     List<SourceSpanSeverityException> errors = [];
     for (var parameter in parameters) {
+      var type = parameter.type;
+      if (type.isDartAsyncFuture) {
+        errors.add(SourceSpanSeverityException(
+          'The type "Future" is not a supported endpoint parameter type.',
+          parameter.span,
+        ));
+        continue;
+      }
```

Comment:
Suggestion: This is a little opinionated but I generally dislike the `continue` keyword, it breaks the flow of the code and gives the same vibes as `break` or `goto`. It can make it harder to understand the code IMO.

I would refactor this in such a way to avoid the need for the continue; keyword. Such as using .map, or .expand, etc.